### DISCUSSION TITLE
parcel-bundler build not working

### DISCUSSION THREAD

**Discussion by hippiemeraki** [parcel-bundler/parcel#6337] [General] [answered]
i am facing an issue while building using parcel build

 in my package.json 
"build-client": "parcel build client/src/index.html --out-dir client/dist"

and the error is

 parcel build client/src/index.html --out-dir client/dist

C:\Users\user\node_modules\bindings\bindings.js:126
  err = new Error(
        ^

Error: Could not locate the bindings file. Tried:
 → C:\Users\user\node_modules\deasync\build\deasync.node
 → C:\Users\user\node_modules\deasync\build\Debug\deasync.node
 → C:\Users\user\node_modules\deasync\build\Release\deasync.node
 → C:\Users\user\node_modules\deasync\out\Debug\deasync.node
 → C:\Users\user\node_modules\deasync\Debug\deasync.node
 → C:\Users\user\node_modules\deasync\out\Release\deasync.node
 → C:\Users\user\node_modules\deasync\Release\deasync.node
 → C:\Users\user\node_modules\deasync\build\default\deasync.node
 → C:\Users\user\node_modules\deasync\compiled\16.1.0\win32\x64\deasync.node
 → C:\Users\user\node_modules\deasync\addon-build\release\install-root\deasync.node
 → C:\Users\user\node_modules\deasync\addon-build\debug\install-root\deasync.node
 → C:\Users\user\node_modules\deasync\addon-build\default\install-root\deasync.node
 → C:\Users\user\node_modules\deasync\lib\binding\node-v93-win32-x64\deasync.node
    at bindings (C:\Users\user\node_modules\bindings\bindings.js:126:9)
    at Object.<anonymous> (C:\Users\user\node_modules\deasync\index.js:30:31)
    at Module._compile (C:\Users\user\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (C:\Users\user\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\Users\user\node_modules\parcel-bundler\src\utils\syncPromise.js:1:79)
    at Module._compile (C:\Users\user\node_modules\v8-compile-cache\v8-compile-cache.js:192:30) {
  tries: [
    'C:\\Users\\user\\node_modules\\deasync\\build\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\build\\Debug\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\build\\Release\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\out\\Debug\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\Debug\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\out\\Release\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\Release\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\build\\default\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\compiled\\16.1.0\\win32\\x64\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\addon-build\\release\\install-root\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\addon-build\\debug\\install-root\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\addon-build\\default\\install-root\\deasync.node',
    'C:\\Users\\user\\node_modules\\deasync\\lib\\binding\\node-v93-win32-x64\\deasync.node'
  ]
}

**Comment by DeMoorJasper** [comment, marked as answer]
This is an issue with deasync: https://github.com/abbr/deasync we no longer use it in parcel 2

deasync has no prebuild binaries for node 16 yet
